scripts/commands/addkeyitem.lua

```diff
@@ -11,32 +11,41 @@ cmdprops =
     parameters = "ss"
 };
 
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("@addkeyitem <ID> <player>");
+end;
+
 function onTrigger(player, keyId, target)
-    if (keyId == nil or keyId == 0) then
```

Comment:
You got a server crash? I didn't get any server crash passing bs strings to the command before when the "or" was added but either when I tried to handle casing or when demo `_G`'d everything we could have broken it I guess.